libs/template/templates/dbt-sql/template/{{.project_name}}/resources/{{.project_name}}_job.yml.tmpl

```diff
@@ -25,9 +21,17 @@ resources:
             # The default schema, catalog, etc. are defined in ../dbt_profiles/profiles.yml
             profiles_directory: dbt_profiles/
             commands:
+{{- if (regexp "^yes").MatchString .personal_schemas}}
+            # The dbt commands to run (see also dbt_profiles/profiles.yml; dev_schema is used in the dev profile)
             - 'dbt deps --target=${bundle.target}'
-            - 'dbt seed --target=${bundle.target} --vars "{ dev_schema: {{$dev_schema}} }"'
-            - 'dbt run --target=${bundle.target} --vars "{ dev_schema: {{$dev_schema}} }"'
+            - 'dbt seed --target=${bundle.target} --vars "{ dev_schema: ${workspace.current_user.short_name} }"'
+            - 'dbt run --target=${bundle.target} --vars "{ dev_schema: ${workspace.current_user.short_name} }"'
+{{- else}}
+            # The dbt commands to run (see also the dev/prod profiles in dbt_profiles/profiles.yml)
+            - 'dbt deps --target=${bundle.target}'
+            - 'dbt seed'
+            - 'dbt run'
```

Comment:
Is the target not required after running deps?

Reply:
Ah yes, it is. Good catch. As you can see it is used in the other branch of the `{{if}}`.